README.md

```diff
@@ -1,16 +1,16 @@
 # Android Food Scanner App 🍌
 
-*The native, Android [Food Scanner App](https://play.google.com/store/apps/details?id=com.lifyzer) 🥝*
+<div align="center" markdown="1">The native, Android [Food Scanner App](https://play.google.com/store/apps/details?id=com.lifyzer) 🥝*
 
-![Healthy Food, Healthy Tomato](extras/assets/broccoli-healthy-food.svg)
+![Healthy Food, Healthy Tomato](extras/assets/broccoli-healthy-food.svg)</div>
 
 
 ## About The App 🤔
 
-* Lifyzer App uses OCR and barcode scanner technologies.
+<div align="center" markdown="1">Lifyzer App uses OCR and barcode scanner technologies.
```

Comment:
Oh... didn't know the `markdown` GitHub's attribute. But not sure it actually does something...

Reply:
according to the preview, it does :smile: Actually, I knew that from MarkdownExtra, and was surprised GH-flavored uses it as well.